Replace InputEvent with Ref<InputEvent> on _input docs

The new docs for _input in the godot-derive portion say to use an InputEvent when using the _input function
https://github.com/godot-rust/godot-rust/blob/master/gdnative-derive/src/lib.rs#L313
```rust
fn _input(&self, owner: &Node, event: InputEvent);
```
But when I do that I get the error that FromVariant isn't implemented for InputEvent
According to one of the Examples in the README one should be using a Ref<InputEvent> when using _input and when I changed it in my code to Ref<InputEvent> it worked
If it should be Ref<InputEvent> that should probably also be reflected in the docs
If it should usually be InputEvent I'd be curious as to why it didn't work in my case

For me personally using &InputEvent doesn't work, says that FromVariant hasn't been implemented, just like when I don't use &
```
cargo build --target x86_64-unknown-linux-gnu 
   Compiling synthstar v0.1.0 (/home/spook/projects/synthstar/rust)
error[E0277]: the trait bound `&gdnative::prelude::InputEvent: FromVariant` is not satisfied
    --> rust/src/game.rs:44:8
     |
44   |     fn _input(&self, _owner: &Node, event: &InputEvent) {
     |        ^^^^^^ the trait `FromVariant` is not implemented for `&gdnative::prelude::InputEvent`
     |
note: required by `from_variant`
    --> /home/spook/.cargo/registry/src/github.com-1ecc6299db9ec823/gdnative-core-0.9.3/src/core_types/variant.rs:1005:5
     |
1005 |     fn from_variant(variant: &Variant) -> Result<Self, FromVariantError>;
     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     = note: this error originates in the macro `$crate::godot_wrap_method_inner` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0277`.
error: could not compile `synthstar` due to previous error
```

Okay so I tried using a reference again with master and now it wants a lifetime parameter
```rust
cargo build --target x86_64-unknown-linux-gnu 
   Compiling synthstar v0.1.0 (/home/spook/projects/synthstar/rust)
error[E0106]: missing lifetime specifier
  --> rust/src/game.rs:44:44
   |
44 |     fn _input(&self, _owner: &Node, event: &InputEvent) {
   |                                            ^ expected named lifetime parameter
   |

For more information about this error, try `rustc --explain E0106`.
error: could not compile `synthstar` due to previous error
```

--- a/gdnative-derive/src/lib.rs
+++ b/gdnative-derive/src/lib.rs
@@ -289,7 +289,7 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// <br><br>
 ///
 /// ```ignore
-/// fn _get_configuration_warning(&self, owner: &Node);
+/// fn _get_configuration_warning(&self, owner: &Node) -> GodotString;
 /// ```
 /// The string returned from this method is displayed as a warning in the Scene Dock if the script that overrides it is a tool script.  
 /// _See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-get-configuration-warning) for more information._
@@ -310,21 +310,21 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// <br><br>
 ///
 /// ```ignore
-/// fn _input(&self, owner: &Node, event: InputEvent);
+/// fn _input(&self, owner: &Node, event: Ref<InputEvent>);
 /// ```
 /// Called when there is an input event.  
 /// _See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-input) for more information._
 /// <br><br>
 ///
 /// ```ignore
-/// fn _unhandled_input(&self, owner: &Node, event: InputEvent);
+/// fn _unhandled_input(&self, owner: &Node, event: Ref<InputEvent>);
 /// ```
 /// Called when an `InputEvent` hasn't been consumed by `_input()` or any GUI.  
 /// _See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-unhandled-input) for more information._
 /// <br><br>
 ///
 /// ```ignore
-/// fn _unhandled_key_input (&self, owner: &Node, event: InputKeyEvent);
+/// fn _unhandled_key_input (&self, owner: &Node, event: Ref<InputKeyEvent>);
 /// ```
 /// Called when an `InputEventKey` hasn't been consumed by `_input()` or any GUI.  
 /// _See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-unhandled-key-input) for more information._
@@ -349,7 +349,7 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// <br><br>
 ///
 /// ```ignore
-/// fn _gui_input(&self, owner: &Control, event: InputEvent);
+/// fn _gui_input(&self, owner: &Control, event: Ref<InputEvent>);
 /// ```
 /// Use this method to process and accept inputs on UI elements.  
 /// _See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_control.html#class-control-method-gui-input) for more information._

